staging/src/k8s.io/apimachinery/pkg/apis/meta/internalversion/types.go

```diff
@@ -66,6 +66,21 @@ type ListOptions struct {
 	// it does not recognize and will return a 410 error if the token can no longer be used because
 	// it has expired.
 	Continue string
+
+	// SendInitialEvents when set together with Watch option, begins the watch stream with synthetic "Added" events of all resources.
```

Comment:
The semantics described here make sense to me.  I think we should add logic that rejects this query param now, similar to what we did with dryrun.  That's a good start even if this doesn't make 1.25 and it gives us a clean PR to backport to supported versions to ease the transition.

Reply:
That makes a lot of sense to me - agree with that.

Reply:
moved to https://github.com/kubernetes/kubernetes/pull/111530